tests/fixtures.py

```diff
@@ -55,9 +55,15 @@ def create_file_after_timeout(path, timeout):
 
 
 def access_self():
+    assert get_current_connection() is not None
     assert get_current_job() is not None
 
 
+def schedule_access_self():
```

Comment:
Since this function is only used in one place, could you please move this function definition to within `test_run_scheduled_access_self`?